src/time_travel/time_machine_clock.py

```diff
@@ -0,0 +1,33 @@
+"""A single clock for all patchers."""
+
+
+class TimeMachineClock(object):
```

Comment:
Come to think of it, I'm not sure the appropriate term in this class is `timestamp`. A timestamp is the time in which something has happened. I think that it would be better to use `clock.time += 5`, and not `clock.timestamp += 5`.

It might also be handy to create another class for the time itself, that way you could override the `=`, `+=`, `-=` operators.